tests/testthat/test_partition_function.R

```diff
@@ -1,9 +1,12 @@
 context("Testing computation of partition functions")
 
+source("../permfun.R")
```

Comment:
I think this line would be unnecessary if `permutations()` is transferred to R/misc.R.